### PR TITLE
Update Mobile-Apps.md

### PR DESCRIPTION
**Pull request by bwromanski**
Updated with a web link.

### CODE REVIEW

File: Mobile-Apps.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   Disadvantages of Hybrid Apps: Limited functionality to take advantage of advanced smartphone hardware features
   Timeframe/Cost: Simple App 3-6 months, $10K
-Cross Platform:
+#Cross Platform:
   Framework Examples: Appcelerator (java), Xamarin (.net, c#)
   Advantages of Cross Platform Apps: Single code base for all platforms, most cost effective for multiple platforms, deep hardware integration, compiles to native, fairly quick to market, great for corporate type applications.
   Disadvantages of Cross Platform Apps: Might require licensing fees, less flexibility with UI/UX as compared to native, lag time for native hardware features, not good for games.
   Timeframe/Cost: Simple App 6 months, $10-$50k
-Native:
+#Native:
```
Comment (vnvemuri) [inline Mobile-Apps.md]: @bwromanski Would be great if you can take a use case of a new app that needs to be developed and then list some of the considerations that you would take into account in order to decide between Hybrid / Cross Platform / Native solutions.

**Comment by ggallicc** [comment]
:+1:

**Comment by ggallicc** [comment]
@PeterBell I cannot merge this branch because it has conflicts. I did some research online and downloaded the Github application but cannot make progress to merge it. Is there anything else we can do?

**Comment by PeterBell** [comment]
Hey @ggallicc add me as a collaborator and I'l resolve the conflicts . ..  let me know when you've done by using an @ mention here...

**Comment by vnvemuri** [comment]
:+1:

**Comment by ggallicc** [comment]
@PeterBell I just added you as collaborator. Thanks a lot!!

**Comment by PeterBell** [comment]
Closing this PR - things break in Git if you create a branch name that is the same as the name of a file. @ggallicc

**Pull request closed without merging**